qemu/tests/rng_hotplug.py

```diff
@@ -4,23 +4,21 @@
 from virttest import error_context
 from virttest.qemu_devices import qdevices
 from virttest import utils_test
-from avocado.core import exceptions
 
 
 @error_context.context_aware
 def run(test, params, env):
     """
     Test hotplug/unplug of rng device
     1) Boot up w/ one rng device
-    2) Unplug rng device
-    3) reboot/shutdown guest(optional)
+    2) Run random read test
+    3) Unplug rng device
```

Comment:
boot guest with one rng device ---> unplug this device, and then repeat hotplug/unplug.
I think you can boot guest without rng device, then start hotplug/unplug loop directly.

Reply:
The step you mentioned is the original implement of virtio-rng hotplug/unplug. IMHO, it's better to open another automation bug for optimization. Thanks!